AI/modelcloud/README.md

```diff
@@ -0,0 +1,148 @@
+# modelcloud example
+
+## Goal: Production-grade inference on AI-conformant kubernetes clusters
+
+This goal of this example is to build up production-grade inference
+on AI-conformant kubernetes clusters.
+
+We (aspirationally) aim to demonstrate the capabilities of the AI-conformance
+profile.  Where we cannot achieve production-grade inference, we hope to
+motivate discussion of extensions to the AI-conformance profile to plug those gaps.
+
+## Walkthrough
+
+### Deploying to a kubernetes cluster
+
+Create a kubernetes cluster, we currently test with GKE and gcr.io but do not aim
+to depend on non-conformant functionality; PRs to add support for deployment
+to other conformant environments are very welcome.
+
+1. From the modelcloud directory, run `dev/tools/push-images` to push to `gcr.io/$(gcloud config get project)/...`
+
+1. Run `dev/tools/deploy-to-kube` to deploy.
+
+We deploy two workloads:
+
+1. `blob-server`, a statefulset with a persistent volume to hold the model blobs (files)
+
+1. `gemma3`, a deployment running vLLM, with a frontend go process that will download the model from `blob-server`.
```

Comment:
Do we need to merge this with https://github.com/kubernetes/examples/tree/master/AI/vllm-deployment? The other one doesn't provide persistent model storage.